transport/protocol/src/lib.rs

```diff
@@ -160,6 +160,20 @@ where
 {
     let mut processes = HashMap::new();
 
+    #[cfg(all(feature = "prometheus", not(test)))]
+    {
+        // Initialize the lazy statics here
+        lazy_static::initialize(&METRIC_RECEIVED_ACKS);
+        lazy_static::initialize(&METRIC_SENT_ACKS);
+        lazy_static::initialize(&METRIC_TICKETS_COUNT);
+        lazy_static::initialize(&METRIC_PACKET_COUNT);
+        lazy_static::initialize(&METRIC_PACKET_COUNT_PER_PEER);
+        lazy_static::initialize(&METRIC_REPLAYED_PACKET_COUNT);
+        lazy_static::initialize(&METRIC_REJECTED_TICKETS_COUNT);
+        lazy_static::initialize(&METRIC_QUEUE_SIZE);
+        lazy_static::initialize(&METRIC_MIXER_AVERAGE_DELAY);
+    }
+
```

Comment:
Can this not be done outside the function?

Reply:
Where ?

Reply:
It has to be somewhere in the code path, it cannot be ran in the "static" section.